src/main/java/frc/robot/RobotContainer.java

```diff
@@ -115,7 +115,10 @@ private void configureButtonBindings() {
 
     //driver controller
     driverControllerClimberButtons.whenActive(new ClimberReleaseSafe(intake, cage, climber, rightCatapult, leftCatapult));
-    driverControllerClimberButtons.whenActive(slowDrive);
+    driverControllerClimberButtons.whenActive(
+      slowDrive
+      .withInterrupt(()-> {return !climber.getIsExtended();})
```

Comment:
@python-42 Since the test did not work today. We could try slow drive combined (e.g SequentialCommandGroup) with a wait command before it so that it won’t get immediately interrupted